Dockerfile

```diff
@@ -1,45 +1,47 @@
-FROM perl:5.38
-
-RUN apt-get update && apt-get install -y \
-  bsd-mailx \
-  msmtp-mta \
-  pigz \
-  ruby-dev
-
-RUN cpanm -n  \
-  Data::Dumper \
-  Date::Manip \
-  DBD::MariaDB \
-  DB_File \
-  DBI \
-  Devel::Cover \
-  Devel::Cover::Report::Coveralls \
-  Exporter \
-  File::Slurp \
-  https://github.com/hathitrust/progress_tracker.git@v0.9.0 \
-  JSON::XS \
-  LWP::Simple \
-  Mail::Mailer \
-  MARC \
-  MARC::Batch \
-  MARC::File::XML \
-  MARC::Record \
-  MARC::Record::MiJ \
-  Sys::Hostname \
-  Test::More \
-  Test::Output \
-  URI::Escape \
-  XML::LibXML \
-  XML::LibXSLT \
-  YAML \
-  YAML::XS
+FROM debian:bookworm
 
 ENV ROOTDIR /usr/src/app
 ENV HOME /usr/src/app
 
 COPY . $ROOTDIR
 WORKDIR $ROOTDIR
 
+# Install Debian packages
+RUN apt-get update && apt-get install -y \
+    bsd-mailx \
+    build-essential \
```

Comment:
I would be curious if we could get away without `build-essential` or if one of the remaining CPAN modules or ruby gems still requires `make` and `gcc` and friends

Reply:
works fine without